frontend/collaborator.tsx

```diff
@@ -1,8 +1,9 @@
-import { Data } from "./data";
 import styles from "./collaborator.module.css";
 import { useEffect, useState } from "react";
 import { Rect } from "./rect";
 import { useCursor } from "./smoothie";
+import { Replicache } from "replicache";
+import { M } from "./mutators";
```

Comment:
import type and use longer name?

Reply:
Huh, I wonder why ts didn't complain about that.

I am going to try on the short name for size. In applications, `Replicache<mutator-type>` is all over the place, in every file. I think a short name is justified.